### DISCUSSION TITLE
HTTP Redirects in nuclei

### DISCUSSION THREAD

**Discussion by r3dpars3c** [projectdiscovery/nuclei#1404] [Q&A] [answered]
There is no flag allowing nuclei to follow redirect like 301 and 302.
Can you please solve this please

**Comment by ehsandeep** [comment, marked as answer]
@Aaryan9898 please use [GitHub discussion](https://github.com/projectdiscovery/nuclei/discussions/categories/q-a) for the questions.

Redirects are not supported with flags, as nuclei works with templates, you can control/define redirects in the template itself - https://nuclei.projectdiscovery.io/templating-guide/protocols/http/, check out [existing templates](https://github.com/projectdiscovery/nuclei-templates/search?q=redirects%3A+true) for example.

**Reply by bgsec** [reply]
Is there any easy way to exclude all redirect templates ?
